app/com/yetu/youtube/controllers/YoutubeController.scala

```diff
@@ -35,7 +37,15 @@ class YoutubeController @Inject() (implicit val env: Environment[User, SessionAu
   def index = SecuredAction { implicit request =>
 
 
-    Ok(views.html.index(FrontendConfiguration.devToken)(FrontendConfiguration.authServerUrl)(FrontendConfiguration.sessionPollingInterval))
+    val config = FrontendConfig(userUUID = request.identity.loginInfo.providerKey,
+    youtubeDeveloperToken = FrontendConfiguration.devToken,
+    authServer = FrontendConfiguration.authServerUrl,
+    sessionPollingInterval = FrontendConfiguration.sessionPollingInterval)
+
```

Comment:
Is this indent established by Scala developers? I'd rather like to read the previous lines as following:
```
val config = FrontendConfig(
    userUUID = request.identity.loginInfo.providerKey,
    youtubeDeveloperToken = FrontendConfiguration.devToken,
    authServer = FrontendConfiguration.authServerUrl,
    sessionPollingInterval = FrontendConfiguration.sessionPollingInterval)
```